src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -563,12 +563,12 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         _lastHoveredCell = terminalPosition;
         uint16_t newId{ 0u };
         // we can't use auto here because we're pre-declaring newInterval.
-        decltype(_terminal->GetHyperlinkIntervalFromPosition(COORD{})) newInterval{ std::nullopt };
+        decltype(_terminal->GetHyperlinkIntervalFromPosition({})) newInterval{ std::nullopt };
         if (terminalPosition.has_value())
         {
             auto lock = _terminal->LockForReading(); // Lock for the duration of our reads.
-            newId = _terminal->GetHyperlinkIdAtPosition(terminalPosition->to_win32_coord());
-            newInterval = _terminal->GetHyperlinkIntervalFromPosition(terminalPosition->to_win32_coord());
+            newId = _terminal->GetHyperlinkIdAtPosition(*terminalPosition);

```

Comment:
Why didn't clang-format catch the extra spaces on this and the next line?

Reply:
There are extra spaces? I don't see them here on GitHub. 🤔

Reply:
<img width="310" alt="image" src="https://user-images.githubusercontent.com/189190/171511784-b307f03f-e41a-4962-b86e-60583358c568.png">

**WHAT**


> `newId = _terminal->GetHyperlinkIdAtPosition(*terminalPosition);`